data/scrape/lib/languages.json

```diff
@@ -460,7 +460,8 @@
             "latn": "Latin",
             "hira": "Hiragana",
             "bopo": "Bopomofo",
-            "hani": "Han"
+            "hani": "Han",
```

Comment:
Please run `git checkout -- languages.json` (in this directory, etc.) on this file. It doesn't need checked in or modified here, even if the procedures caused it to be. (I think that's a very minor bug for us to fix separately.)